app/experimenter/docs/adrs/0002-use-react-bootstrap-cra-gql-for-nimbus.md

```diff
@@ -0,0 +1,113 @@
+* Continue using Bootstrap (`react-bootstrap`), but:
+  * Add stylelint for SCSS linting overrides
+  * Use breakpoints established in FxA
+* Utilize Storybook for browsing the UI in various states without running the stack
+* Add Sentry to catch client-side errors
+
+## Considered Alternatives
+
+* Refactor existing `rapid` project rather than use CRA for `nimbus-ui`
+* Use existing API calls instead of switching to GQL
+* Use Tailwind instead of Bootstrap
+
+### Positive Consequences
+
+Continuing to use React and Bootstrap while starting a new app with `create-react-app` and GraphQL/Apollo allows us to benefit from stack and tooling improvements while avoiding an entire rewrite of all front-end components.
+
+Some members of the FxA team are rolling off of a React/GQL project, `fxa-settings`, with a similar front-end stack and [good documentation](https://github.com/mozilla/fxa/tree/main/packages/fxa-settings). This project can be referenced for using CRA, Storybook examples, how to set up and use the apollo cache, testing helpers, and testing examples using GQL queries and mutations.
+
+### Negative Consequences
+
+This approach for `nimbus-ui` will require time to reach parity to the current `rapid` project. 
+
+## Pros and Cons of the Considered Alternatives
+
+### Refactor existing project
+
+* Pros
+  * Less time spent on new project scaffolding
+* Cons
+  * Because the `rapid` project is small, this may cause points of frustration and more time than expected shuffling files around, refactoring, modifying tests, and reconnecting the pieces with a new global state management approach compared to starting anew
+  * We desire to use the build pipeline offered and managed by `create-react-app` regardless (see [FxA's React Toolchain ADR](https://github.com/mozilla/fxa/blob/main/docs/adr/0013-react-toolchain-for-settings-redesign.md)) and it may be awkward porting an existing application over
+  * Refactoring instead of "starting fresh" provides a weaker sense of collective ownership from new contributors and excitement for fresh perspectives
+
+### Don't use GQL
+
+* Pros
+  * Less additional work on the front-end and back-end initially
+  * Non-FxA engineers would likely experience a learning curve switching to GQL
+* Cons
+  * As the application grows, we will likely need to set up Redux or grow the current ad-hoc global state management
+  * We won't reap the benefits of a GQL application (see (FxA's GQL ADR)[https://github.com/mozilla/fxa/blob/main/docs/adr/0016-use-graphql-and-apollo-for-settings-redesign.md]) such as reduced network requests and data transferred, clarity around expected data and server responses, client-side caching and global state management via `apollo-client-cache`, helpful developer tooling, and speedier development in the long run
+
+### Use Tailwind instead of Bootstrap
+
+* Bootstrap
+  * Pros
```

Comment:
\+ Bootstrap's [utility classes](https://getbootstrap.com/docs/4.5/utilities/), if we should choose to use them, are somewhat similar to Tailwind classes (just a little more verbose). Reduced learning curve. There are also a multitude of IDE extensions (like [VS Code](https://marketplace.visualstudio.com/items?itemName=Zaczero.bootstrap-v4-snippets)).